internal/future/serving/server.go

```diff
@@ -45,6 +47,11 @@ type Params struct {
 	handlersForGrpcProxy []GrpcProxyHandler
 	grpcListener         *netlistener.ListenerHolder
 	grpcProxyListener    *netlistener.ListenerHolder
+
+	// TLS configuration
+	rootCaPublicCertificateFileData []byte
+	publicCertificateFileData       []byte
+	privateKeyFileData              []byte
 }
 
 // NewParamsFromConfig returns.
```

Comment:
improve comment? Similar for other places in the file that have placeholder comments.

Reply:
I was referring to the following function comments.
// NewParamsFromConfig returns.
// NewParamsFromListeners returns.